File: app/src/main/java/im/tox/antox/callbacks/AntoxOnVideoDataCallback.java
package im.tox.antox.callbacks;

import android.content.Context;
import android.util.Log;

import im.tox.antox.utils.AntoxFriend;
import im.tox.jtoxcore.callbacks.OnVideoDataCallback;

/**
 * Created by Mark Winter on 03/09/14.
 */
public class AntoxOnVideoDataCallback implements OnVideoDataCallback<AntoxFriend> {

    private Context ctx;

    public AntoxOnVideoDataCallback(Context ctx) {
        this.ctx = ctx;
    }

    public void execute(int callID, byte[] data, int width, int height) {
        Log.d("OnVideoDataCallback", "Received a callback from: " + callID);
    }
}

File: app/src/main/scala/im/tox/antox/callbacks/AntoxOnVideoDataCallback.scala
package im.tox.antox.callbacks

import android.content.Context
import android.util.Log

class AntoxOnVideoDataCallback(private var ctx: Context) {

  def execute(callID: Int,
    data: Array[Byte],
    width: Int,
    height: Int) {
    Log.d("OnVideoDataCallback", "Received a callback from: " + callID)
  }
}
